Extensions/xkit_updates.js

```diff
@@ -468,6 +468,39 @@ XKit.extensions.xkit_updates = new Object({
 
 	},
 
+	shouldUpdate: function(versionCurrent, versionRemote) {
+		if (versionRemote.major > versionCurrent.major) {
+			return true;
+		}
+		if ((versionRemote.minor > versionCurrent.minor) && (versionRemote.major <= versionCurrent.major)) {
```

Comment:
I think `versionRemote.major <= versionCurrent.major` should be `versionRemote.major === versionCurrent.major`.  We don't want to update if the current version is 1.0.0 and the remote version is 0.2.0.